Sim/SimDelphesInterface/src/DelphesSaveChargedParticles.cpp

```diff
@@ -50,6 +81,33 @@ StatusCode DelphesSaveChargedParticles::saveOutput(Delphes& delphes, const fcc::
     auto cand = static_cast<Candidate*>(delphesColl->At(j));
     auto particle = colParticles->create();
 
+
+    auto cov_d = CovToACTS(cand->TrackCovariance, cand->CtgTheta, cand->C);
+    auto t = colParticles_trkCov->create();
+    t.d0(1000* cand->D0);
+    t.z0(1000* cand->DZ);
+    t.phi(cand->Phi);
+    t.theta(TMath::ATan(1.0 / cand->CtgTheta) + TMath::PiOver2());
```

Comment:
there is a bug here, it should be:
`t.theta(TMath::ATan2(1.0,cand->CtgTheta);`		// Theta in [0, pi] range
(just checked with Bedeschi and Alcatraz)